DEPS.bzl

```diff
@@ -3923,10 +3923,6 @@ def go_deps():
         name = "com_github_golang_protobuf",
         build_file_proto_mode = "disable_global",
         importpath = "github.com/golang/protobuf",
-        patch_args = ["-p1"],
-        patches = [
```

Comment:
Why is this being dropped? What was in that patch?

Reply:
[This](https://github.com/cockroachdb/cockroach/pull/133497/files#diff-7c7c10317a1d8f3e345af180d2a1af185e0c3cb33419385b423a08ef5f96f157) is the patch in question. It's a low-level build thing that replaces one dependency with another to correct a build breakage. But, it's no longer necessary with the update.